doc/nrf/ug_gzp.rst

```diff
@@ -0,0 +1,298 @@
+|                |         |       |                                                      | Generated when first Host ID request is received.     |
++----------------+---------+-------+------------------------------------------------------+-------------------------------------------------------+
+
+The System Address and Host ID are stored in non-volatile memory (NVM) and apply for the lifetime of the Host unless the NVM is erased.
+
+.. figure:: images/gzp_factory_defaults.svg
+   :alt: Initial pairing parameters
+
+   Initial pairing parameters
+
+.. _ug_gzp_sys_addr_exchange:
+
+System Address exchange
+=======================
+
+The System Address is an individual Host's private address.
+For a Device to communicate with an individual Host, it needs to know the Host's System Address.
+When requesting a System Address, the Device sets the TX power to a low level using GZP_POWER.
+By measuring the received signal strength when a pairing request is received from a Device, the Host can estimate the relative proximity of the requesting Device.
+
+For a Device to pair with a Host, it must first obtain the System Address on which all subsequent key exchange and data transfer occur.
+This transaction occurs on the pipe 0 and is transmitted in cleartext on the air as it is not a secret.
+
+.. figure:: images/gzp_address_exchange.svg
+   :alt: System address exchange
+
+   System Address exchange
+
+.. _ug_gzp_host_id_exchange:
+
+Host ID exchange
+================
+
+Once the Device has the System Address, it can request the Host ID on the pipe using GZP_DATA_PIPE.
+The Host ID is used to generate subsequent "dynamic keys" for encrypted data transfer.
+
+After receiving a Host ID request, the Host generates the ID if it has not done it already.
+The Host ID is generated using the random Session Token received from the Device in the Host ID request as well as the session counter.
+
+The Device sends a packet to fetch the Host ID.
+The secret Host ID is transmitted on the encrypted pipe GZP_DATA_PIPE using the shared Secret Key.
+
+The following security precautions are taken for the Host ID exchange:
+
+* Passive eavesdropping is prevented by using AES encryption.
+* Replay attacks are prevented by using session tokens.
+* Man-in-the-middle and malicious device attacks can be prevented by implementing a user validation stage before the Host ID is sent to the Device (see :ref:`gazell_host_validation`).
+
+.. figure:: images/gzp_host_id_exchange.svg
+   :alt: Host ID exchange
+
+   Host ID exchange
+
+The Host ID can be compromised if the attacker has knowledge of the Secret Key.
+The attacker could eavesdrop the Host ID exchange and obtain the Host ID or attempt to pair as a malicious device and obtain the Host ID.
+Before the Host ID is sent from the Host to the Device, the application can add a validation stage.
+
+.. _gazell_host_validation
```

Comment:
```suggestion
.. _gazell_host_validation:
```
My bad, sorry.

Reply:
The label is renamed and a colon is added.

Reply:
When you rename the label, you also need to change the link to this label. I commented this accordingly where the actual link is.